receiver/udp.go

```diff
@@ -56,6 +63,9 @@ func (ud *UDP) process() {
 // listening for incoming UDP messages on the configured address. Start
 // never returns.
 func (ud *UDP) Start() error {
+	if ud.PacketSize < 0 || ud.PacketSize > UDP_MAX_READ_SIZE {
```

Comment:
Read my note below, and after that is fixed - I think a PacketSize  of 0 should mean "use default", and the default can probably be 9000.
This also has the benefit of test cases not needing to be touched as much, and it keeps backward compatibility.